website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Multi-Tenancy Best Practices"
+linkTitle: "Multi-Tenancy"
+weight: 1
+description: This guide describes the best practices concepts to write Operators for Multi-Tenancy solutions.
+---
+
+## Operator Behavioral Guidelines
+
+### Create resource requests
+
+* for your Operator's own `Pods`
+* for `Pods` your operator creates / manages
+
+The reasoning behind this is, to allow [ResourceQuota][k8s-resources-quotas] configuration
+to work. This is something that every production cluster will follow as a best practice, to prevents tenants
+from being able to consume all resources on the cluster and block other worklaods from being scheduled. 
+[ResourceQuota][k8s-resources-quotas] in a for CPU and RAM to be rejected. For further information see:
+[Requests and limits][k8s-requests-limits]
+
+**IMPORTANT:** Because OLM-managed Operators typically don't run on Cluster infra nodes, they are consuming paid CPU cores from a user
+perspective, so they naturally want to at least limit the maximum amount of resources such non-workload components can take up
```

Comment:
```suggestion
**IMPORTANT:** Because OLM-managed Operators do not run on Cluster infra nodes, they will consume paid CPU cores. Users will naturally want to limit the maximum amount of resources these non-workload components can take up.
```